Adds content to toolbar examples

edonehoo opened the pull request:
Makes progress on https://github.com/patternfly/patternfly-org/issues/2990

patternfly-build commented:
Preview: https://patternfly-react-pr-8906.surge.sh

nicolethoen commented:
@edonehoo 
There is an accessibility test failure because each of our toolbar examples was assigned an `id` (not quite sure why), and every heading on the page is also given an `id`. The `Toolbar items` heading you added is now conflicting with the id on toolbar in the Toolbar items example. 
If you update the id on [this line](https://github.com/edonehoo/patternfly-react/blob/iss2990-toolbar/packages/react-core/src/components/Toolbar/examples/ToolbarItems.tsx#L22) to something like `'toolbar-items-example'` then the violation should be resolved.

mcarrano reviewed (changes requested):
It feels odd to me that Sticky Toolbar is the first example. It was not previously. Is there are reason we changed the order?

Also, @nicolethoen , I assume it's unrelated to this PR, but do we understand why we are see a vertical mis-alignment now of certain items in toolbar examples?

thatblindgeye commented on packages/react-core/src/components/Toolbar/examples/Toolbar.md:
```diff
@@ -23,90 +23,112 @@ KebabToggle
 ### With custom chip group content
 
-The chip groups generated by toolbar filters may be further customized through the `customChipGroupContent` property, which will append to the filter chip groups. This property will remove the default `Clear all filters` button.
+To customize the chip groups generated by toolbar filters, use the `customChipGroupContent` property. This property will remove the default `clear all filters` button.
```
```suggestion
To customize the chip groups generated by toolbar filters, use the `customChipGroupContent` property on the `<Toolbar>`. This property will remove the default `clear all filters` button.
```

nicolethoen commented:
> Also, @nicolethoen , I assume it's unrelated to this PR, but do we understand why we are see a vertical mis-alignment now of certain items in toolbar examples?

which examples? I am not seeing misalignment when I preview the toolbar examples

mcarrano commented:
@nicolethoen I see the problem here, for example: https://patternfly-react-pr-8906.surge.sh/components/toolbar#component-managed-toggle-groups

![Screenshot 2023-04-25 at 2 08 26 PM](https://user-images.githubusercontent.com/6619860/234364586-f0f244ca-e95d-4592-ac2b-5e473bb2d97f.png)

I only see this in Firefox. Not in Chrome.

nicolethoen commented:
Ah I am not seeing that on chrome - but i see it on firefox. bummer. We can open a follow up issue for that

nicolethoen commented:
@mcarrano here is the follow up issue to address select alignment in toolbars in firefox: https://github.com/patternfly/patternfly-react/issues/9004

edonehoo commented on packages/react-core/src/components/Toolbar/examples/Toolbar.md:
```diff
@@ -23,90 +23,112 @@ KebabToggle
-### Adjusting item spacers
+### Toolbar item spacers
+
+You may adjust the space between toolbar items to arrange them into groups. [Read our spacers documentation](/v4/guidelines/spacers) to learn more about using spacers.
```
I removed the link for now to avoid the 404, but we can add it back when we figure out where to accurately point it

edonehoo commented:
Regarding "It feels odd to me that Sticky Toolbar is the first example. It was not previously. Is there are reason we changed the order?": I thought that it seemed like the most "basic" example since it referred to the toolbar behavior itself, rather than the toolbar items. But, if it seems odd, that was my only reason. I went ahead and moved it back to its original location assuming that makes better sense overall!

mcarrano commented:
Looks great. Thanks for the update @edonehoo .
